package.json

```diff
@@ -97,16 +97,16 @@
     "react-native-google-places-autocomplete": "git+https://github.com/Expensify/react-native-google-places-autocomplete.git#e12768f1542e7982d90f6449798f0d6b7f18f192",
     "react-native-haptic-feedback": "^1.13.0",
     "react-native-image-pan-zoom": "^2.1.12",
-    "react-native-image-picker": "^5.0.2",
+    "react-native-image-picker": "^5.1.0",
     "react-native-image-size": "git+https://github.com/Expensify/react-native-image-size#6b5ab5110dc3ed554f8eafbc38d7d87c17147972",
     "react-native-modal": "^13.0.0",
-    "react-native-onyx": "1.0.36",
+    "react-native-onyx": "git+https://github.com/Expensify/react-native-onyx.git#4e243460ab9ea7dde1a07cb1093cea4040d7306c",
```

Comment:
Waiting on https://github.com/Expensify/react-native-onyx/pull/234

Reply:
The PR is merged. Can we update the react-native-onyx to 1.0.38, so that we can test with actual change?